### ISSUE TITLE
Tabs - change keyboard behavior to support arrow keys to move focus between tabs

### ISSUE THREAD

**Issue by jcfilben** [grommet/grommet#7962]
The tabs component uses the tablist role. Based on the information here: https://w3c.github.io/aria/#tablist. The tablist role should support moving focus between tabs via the left and right arrow keys.

Currently each tab is it's own focus stop and you have to press the 'tab' key to move between them. The expected behavior should be: tab key moves focus into or out of the tablist and arrow keys move focus within the tablist when it has focus.

This issue was raised by GLP
